src/library/scala/math/Ordering.scala

```diff
@@ -370,17 +370,17 @@ object Ordering extends LowPriorityOrderingImplicits {
 
   /** `Ordering`s for `Float`s.
     *
-    * The behavior of the comparison operations provided by default (implicit)
-    * ordering on `Float` has changed in 2.10.0 and 2.13.0.
-    * Prior to Scala 2.10.0, the `Ordering` instance used the semantics
+    * The behavior of the comparison operations provided by the default (implicit)
+    * ordering on `Float` changed in 2.10.0 and 2.13.0.
+    * Prior to Scala 2.10.0, the `Ordering` instance used semantics
     * consistent with `java.lang.Float.compare`.
     *
-    * Scala 2.10.0 changed the implementation of `lt`, `equiv`, `min` etc to be
-    * IEEE 754 compliant, while keeping `compare` method NOT compliant,
-    * creating an internally inconsistent instance. IEEE 754 specified that
-    * `0.0F == -0.0F`. In addition, all comparisons with `Float.NaN` must return
+    * Scala 2.10.0 changed the implementation of `lt`, `equiv`, `min`, etc., to be
+    * IEEE 754 compliant, while keeping the `compare` method NOT compliant,
+    * creating an internally inconsistent instance. IEEE 754 specifies that
+    * `0.0F == -0.0F`. In addition, it requires all comparisons with `Float.NaN` return
     * `false` thus `0.0F < Float.NaN`, `0.0F > Float.NaN`, and
-    * `Float.NaN == Float.NaN` all yield `false`, analogous to `None`.
+    * `Float.NaN == Float.NaN` all yield `false`, analogous calling `exists` on `None`.
```

Comment:
I am saying that `NaN` represents an error state that propagates like `None` does in `flatMap`. Not sure bringing up `exists` makes that clearer.

Reply:
The mention of `flatMap` works too.  I just thought it was left a little too unclear to just say "analogous to `None`".